src/molecules/modals/PlaceholderSpinner.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
Is there a reason why we wouldn't put that svg into the Icons?

Reply:
@shan8851 ?

Reply:
@goshatravin well I was thinking this maybe a loading spinner here rather than just an image/icon. But yeah if it is the latter I will move to icons as now that I think about this, it probably isn't a loading spinner